sbws/lib/v3bwfile.py

```diff
@@ -43,6 +43,25 @@
 STATS_KEYVALUES = ['number_eligible_relays', 'minimum_number_eligible_relays',
                    'number_consensus_relays', 'percent_eligible_relays',
                    'minimum_percent_eligible_relays']
+BW_HEADER_KEYVALUES_RECENT_MEASUREMENTS_EXCLUDED = [
+    # The number of success results should be:
+    # the number of attempts - the number of failures
+    # 4.6 header: the number of successful results, created in the last 5 days,
+    # that were excluded by a filter
+    # This is the sum of the following 3 + not success results
+    # 'recent_measurement_exclusion_count',
+    'recent_measurement_exclusion_not_success_count',
```

Comment:
This key name is hard to understand, because it uses two negatives in a row: "exclusion not success".

Usually, we use a short name for the total, and then add extra information for each part of the total:
```suggestion
    'recent_measurement_exclusion_count',
```

Reply:
I left commented that key because that would be the sum of all the relays excluded by a rule.
What about `recent_measurement_exclusion_error_count`?

Reply:
This is actually the sum of all the errors, which we can count externally, so maybe just not include it?

Reply:
I think we do not need it. Most parsers will be able to add numbers.

Reply:
hmm, thinking it better, we could now deduce all the relays excluded by a rule counting "vote=0" when it's implemented and then looking why there were measurements excluded for each relay.
But if we leave all the reasons whey the relays are excluded, it'll give us a fast idea on what it's failing.
And then i think we should leave this one for consistency.